internal/encryption/method/unencrypted/method.go

```diff
@@ -19,12 +20,12 @@ func (f *descriptor) ID() method.ID {
 	return "unencrypted"
 }
 func (f *descriptor) ConfigStruct() method.Config {
-	return new(config)
+	return new(mconfig)
 }
 
-type config struct{}
+type mconfig struct{}
```

Comment:
Is this intended to be short for `methodConfig`? Could we just call it `methodConfig`, if so? or... `methodConfigBuilder`, since it seems to have a `Build` method. :thinking: